riptide-spring-boot-autoconfigure/pom.xml

```diff
@@ -97,6 +101,16 @@
             <groupId>org.zalando</groupId>
             <artifactId>riptide-httpclient</artifactId>
         </dependency>
+        <dependency>
+            <groupId>jakarta.xml.soap</groupId>
+            <artifactId>jakarta.xml.soap-api</artifactId>
```

Comment:
<artifactId>jakarta.xml.ws-api</artifactId> should be enough here instead of soap and bind separate artefacts

Reply:
There are compilation errors if I don't include them

Reply:
hm, just verified - it's because spring 5 still depends on Jakarta 2.3.3 (packages javax.xml). So I'm not sure how new changes to Jakarta namespaces will work with spring 5. 

Can you try to move to jdk17 but still use javax.xml for web services as first step?

Reply:
Sure. Will try that tomorrow